savant/app/query.py

```diff
@@ -153,3 +153,22 @@ def new_users(self):
         self.conn.commit()
         
         return self.new_users
+
```

Comment:
Following up on comments on #141, assuming this branch is part of it. Strange, I don't see those changes here. I am confused 😄 

Anyway, following those thoughts, we should not need `null_cases` here.